buildSrc/private/src/main/kotlin/androidx/build/AndroidXComposeImplPlugin.kt

```diff
@@ -138,10 +138,20 @@ class AndroidXComposeImplPlugin : Plugin<Project> {
             }
 
             project.tasks.withType(KotlinJsCompile::class.java).configureEach { compile ->
+                val isWasm = compile.kotlinOptions.freeCompilerArgs.contains("-Xwasm")
```

Comment:
for the nearest release we'll keep decoys enabled for k/js, but not for k/wasm. In the future it will be disabled for both.